src/video/x11/SDL_x11mouse.c

```diff
@@ -373,10 +374,10 @@ static int X11_SetRelativeMouseMode(SDL_bool enabled)
     if (X11_Xinput2IsInitialized()) {
         return 0;
     }
+    return -1;
```

Comment:
error?

Reply:
this seems, ok:
original code was:
https://github.com/libsdl-org/SDL/blob/main/src/video/x11/SDL_x11mouse.c#L370
```
static int X11_SetRelativeMouseMode(SDL_bool enabled)
{
#if SDL_VIDEO_DRIVER_X11_XINPUT2
    if (X11_Xinput2IsInitialized()) {
        return 0;
    }
#else
    SDL_Unsupported();
#endif
    return -1;
}
```
I changed it to:
```
371 static int X11_SetRelativeMouseMode(SDL_bool enabled)
372 {
373 #if SDL_VIDEO_DRIVER_X11_XINPUT2
374     if (X11_Xinput2IsInitialized()) {
375         return 0;
376     }
377     return -1;
378 #else
379     return SDL_Unsupported();
380 #endif
381 }
```

Reply:
It should be:
```c
static int X11_SetRelativeMouseMode(SDL_bool enabled)
{
#if SDL_VIDEO_DRIVER_X11_XINPUT2
    if (X11_Xinput2IsInitialized()) {
        return 0;
    }
#endif
    return SDL_Unsupported();
}
```